boards/luatos/esp32c3_luatos_core/board.yml

```diff
@@ -0,0 +1,9 @@
+boards:
+  - name: esp32c3_luatos_core
+    vendor: luatos
+    socs:
+      - name: esp32c3
+  - name: esp32c3_luatos_core_usb
+    vendor: luatos
+    socs:
+      - name: esp32c3
```

Comment:
why is this two boards ?

Seems to me this ought to be made like this:
```
boards:
  - name: esp32c3_luatos_core
    vendor: luatos
    socs:
      - name: esp32c3
        variants:
        - name: usb
```

which is also what is documented here:
https://github.com/zephyrproject-rtos/zephyr/blob/2851032b1b46aab33f46fe165de36e98107c6e5b/boards/luatos/esp32c3_luatos_core/doc/index.rst?plain=1#L43

(this comment also applies to the luatos board)

Reply:
The `esp32c3_luatos_core` is still implemented as two boards, but documented as `esp32c3_luatos_core/esp32c3`  and the `esp32c3_luatos_core/esp32c3/usb`  variant.